README.md

```diff
@@ -1 +1,41 @@
 # che-workspace-crd-operator
+
+## Development
+
+### Dependencies
+
+Dependencies in the project are managed by Go Dep. After you added a dependency you need to run the following command to download dependencies to vendor repo and lock file and then commit changes:
+
+```
+dep ensure
+```
+
+Note that dep ensure doesn't automatically change Gopkg.toml which contains dependencies constraints. So, when a dependency is introduced or changed it should be reflected in Gopkg.toml.
+
+### Build docker image
+
+Since it uses Go Dep, you need operator-sdk 0.10.0 where Go Modules is not a default vendor.
```

Comment:
You mean version "0.10.0 or later" or that versions other than 0.10.0 won't work? In either case you should be more specific. And I guess that golang version is important here as well.

Reply:
In fact it should *not* be 0.12.0 since the support for `dep` was removed in favor of go modules.
And it was started / implemented with the `0.10.0` version. But maybe using the `0.11.0` can work.
This is why recommending using the `0.10.0` was chosen.

Reply:
Rephrased as `you need exact operator-sdk 0.10.0 version`

I believe a bit later we will move to modules and will be able to change it to `0.12.0 or later` and for time being it's OK.